src/test/scala/viper/gobra/parsing/GoParserUnitTests.scala

```diff
@@ -31,19 +31,19 @@ class GoParserUnitTests extends AnyFunSuite with Matchers with Inside {
     "package p; func f(x ...int,) {};",
     "package p; type T []int; var a []bool; func f() { if a[T{42}[0]] {} };",
     "package p; type T []int; func g(int) bool { return true }; func f() { if g(T{42}[0]) {} };",
-    "package p; type T []int; func f() { for _ = range []int{T{42}[0]} {} };",
+    "package p; type T []int; func f() { for _ = range []int{T{42}[0]} with _ {} };",
```

Comment:
Does this mean that we do not support a range loop without the `with` clause?

Reply:
I would think that to prove the useful stuff you would need the enumerator. Without it you can't even prove termination easily, I can make it optional though.